care.config.ts

```diff
@@ -55,6 +55,8 @@ const careConfig = {
   defaultEncounterType: (env.REACT_DEFAULT_ENCOUNTER_TYPE ||
     "hh") as EncounterClass,
 
+  mapUrl: env.REACT_MAP_URL || "https://maps.google.com/?q=",
```

Comment:
let's keep the default in `.env` instead.

also let's use `{lat}`, `{lng}` in the env string which would be replaced by

```suggestion
  mapUrl: (lat, lng) => env.REACT_MAP_URL.replace(...replace lat and lng with args...),
```

Reply:
or better, just export the map url template in care config and have a util function to have this function, let's not keep functions in care config.

Reply:
Done

Reply:
@rithviknishad , I went with second approach to create utility func do it still need to put in .env file ?